commons/src/test/java/io/hyscale/commons/handler/TailLogHandlerTests.java

```diff
@@ -20,9 +20,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class TestTailLogHandler implements TailHandler {
+public class TailLogHandlerTests implements TailHandler {
```

Comment:
Should we rename it as TailLogTestHandler?
Because this sound like it contains test cases